src/Hook/ViewsViewAlterPostRenderHook.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\coyote_img_desc\Hook;
+
+use Coyote\ContentHelper\Image;
+use Drupal\Core\Render\Markup;
+use Drupal\Core\Security\TrustedCallbackInterface;
+use Drupal\coyote_img_desc\ContentParser;
+use Drupal\coyote_img_desc\Util;
+
+class ViewsViewAlterPostRenderHook implements TrustedCallbackInterface {
+
+  /**
+   * @inheritDoc
+   */
+  public static function trustedCallbacks() {
+    return ['hook'];
+  }
+
+  public static function hook(Markup $markup, array $output): string
```

Comment:
Since this code is almost identical to the implementation of `EntityViewAlterPostRenderHook::hook`, that should probably go into a common helper function.

Reply:
Hmm, EntityView has a few extra checks and then also uses the already common ContentParser::replaceImageDescriptions. 
NodeView... can be dropped.